packages/block-library/src/site-tagline/block.json

```diff
@@ -8,13 +8,11 @@
 	"keywords": [ "description" ],
 	"textdomain": "default",
 	"attributes": {
-		"placeholder": {
-			"type": "string"
-		},
 		"textAlign": {
 			"type": "string"
 		}
 	},
+	"example": {},
```

Comment:
So is this the new canonical way to set block examples or are they only added to `block.json` for certain cases? All the ones I can see in trunk seem to be dynamic blocks.

Reply:
Thanks for reviewing! I wasn't too sure what's the preferred approach, they both seem to work, and I was mostly copying from the example over in https://github.com/WordPress/gutenberg/pull/48326. However, the reason I'd included `example` in `index.js` is that we can wrap any strings in translation calls, so that might be a reason why many of them use `index.js` instead (e.g. Buttons block's example has strings for the text attribute, etc)

@ntsekouras do you have a preference for where `example` goes?

Reply:
I think is better to have properties in `block.json`,  when we can. There have been a lot of work from @gziolo in this area.

Reply:
You can't put everything in `block.json` for `example`, e.g. translatable strings. I don't have whether I have a preference here.